src/helics/shared_api_library/helicsCore.h

```diff
@@ -110,7 +110,8 @@ HELICS_EXPORT HelicsBool helicsIsCoreTypeAvailable(const char* type);
  * @param type The type of the core to create.
  * @param name The name of the core. It can be a nullptr or empty string to have a name automatically assigned.
  * @param initString An initialization string to send to the core. The format is similar to command line arguments.
- *                   Typical options include a broker name, the broker address, the number of federates, etc.
+ *                   Typical options include a broker name, the broker address, the number of federates, etc.  Can also be a
+ file(toml,ini,json) or json object containing the core configuration
```

Comment:
```suggestion
 *                   Typical options include a broker name, the broker address, the number of federates, etc.  Can also be a
 file (toml, ini, json) or json object containing the core configuration.
```

When the docstrings are parsed, is the indentation used to signal that it is part of the description for the previous @param?